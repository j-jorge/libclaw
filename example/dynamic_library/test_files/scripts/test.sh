#!/bin/sh

./ex-dynamic_library in_program nowhere first_function second_function
./ex-dynamic_library nowhere first_function second_function
