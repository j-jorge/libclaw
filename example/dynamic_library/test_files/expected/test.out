in_program : The string passed to a method in the current program
Symbol 'nowhere' not found.
first : The string passed to the method
second : The string passed to the method
Symbol 'nowhere' not found in the current program.
first : The string passed to the method
second : The string passed to the method
